.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/home/arthur/VisualStudioProjects/Python/ranked-voting/venv/bin/python3.8"
```

Comment:
This is your local path, it depends on the user's operation system as well.
If we must keep this file we could use an environment variable? 🤔

Reply:
Ill change the interpreter path... sorry haha

Reply:
Ill try to use the relative path

Reply:
now i took deeper observation of it, and i forgot to add .vscode stuff in the .gitignore, my bad